core/src/trezor/lvglui/i18n/locales/zh_cn.py

```
@@ -1042,11 +1042,14 @@
     "助记词的单词数量并不会影响安全性，所有助记词在加密层面上都是安全的。",
     "显示动态二维码",
     "BTC, TRON, SOL, ETH, BNB ...",
-    "部分加密资产和硬件功能在二维码连接模式下不可用。 \n此连接方式仅适用于极少数不常操作硬件钱包的用户，且无法与其他连接方式兼容。 \n如需要通过蓝牙或 USB 连接硬件钱包，请重新添加钱包以切换连接模式。",
+    "部分加密资产和硬件功能在二维码连接模式下不可用。 \n\n此连接方式仅适用于极少数不常操作硬件钱包的用户，且无法与其他连接方式兼容。 \n\n如需要通过蓝牙或 USB 连接硬件钱包，请重新添加钱包以切换连接模式。",
```

Ruff (0.14.7)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)